### DISCUSSION TITLE
Is it possible to disable scanJobLogs?

### DISCUSSION THREAD

**Discussion by janisii** [aquasecurity/trivy-operator#1467] [Help & Support] [answered]
Hello, 
we are using `trivy-operator` installed with `HELM` on our K8s cluster. We would love to turn off `trivy-operator` to send scan dumps to pods stdout, as we use Loki with promtail and all the dump is being sent to pods logs (scan-vulnerabillityreport-xxxx pod + sidecard).

`trivyOperator.scanJobCompressLogs: true` -> this compress the logs, but still a lot of noise. Is it possible to turn it off completely?

Thanks.

**Comment by chen-keinan** [comment, marked as answer]
@janisii the scan job logs are the actual trivy scanning results , without it,  no report will be generated.

The lag above is only if to compress it or not , as logs might be large and exceed pod limit.
